src/index.js

```diff
@@ -7,6 +7,7 @@ import { BrowserRouter as Router } from "react-router-dom";
 import { VideoProvider } from "./context/vedio-data-context";
 import { HistoryProvider } from "./context/history-context";
 import { LikeProvider } from "./context/like-context";
+import {PlayListProvider} from './context/playlist-context'
```

Comment:
You can import all context from one file

Reply:
Sure, i will update